.mocharc.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
+    extension: 'js',
+    forbidOnly: true,
```

Comment:
This will forbid `mocha` with .only.

Reply:
this was already the case, but we passed the options directly

Reply:
exactly, `mocha` != `npm test`.

Reply:
fixed